include/ignition/common/Console.hh

```diff
@@ -183,6 +183,13 @@ namespace ignition
                    /// \brief Destructor.
                    public: virtual ~Buffer();
 
+                   /// \brief Writes _count characters to the string buffer
+                   /// \param[in] _char Input rharacter array.
+                   /// \param[in] _count Number of characters in array.
+                   /// \return The number of characters successfully written.
+                   public: virtual std::streamsize xsputn(
```

Comment:
I believe that `sync` below also needs to be marked override to make macos happy.